Android/app/src/main/java/io/github/project_travel_mate/utilities/ShareContactActivity.java

```diff
@@ -55,6 +55,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
         ButterKnife.bind(this);
 
+
+
```

Comment:
why this change? Revert this

Reply:
Mam can u pls help me?...m getting <! DOCTYPE html> error during login time ....how can I solve this error